.github/workflows/tf-aws-test.yml

```diff
@@ -0,0 +1,38 @@
+name: SDK Spec AWS Tests
+on:
+  schedule:
+    - cron: "0 8 * * *"
+  workflow_dispatch: {}
```

Comment:
Would this allow any of our contributors with permissions to create branches on our repo to run the workflow on their own branch? Is there a way to scope this down so the workflow can only be executed on protected branches or by certain users?

Reply:
I understand the concern,
I spoke to Elad and we decided to run it upon release/ on PR (when a suitable flag is on), I believe that the contributors (or at least their PRs) will have to have the option to run and re-run those tests... I'll see if there is something to do to restrict the credentials from being abused